apiserver.local.config/certificates/apiserver.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
where do you use those certificates?

Reply:
also why not generating them on the fly if they're used for tests

Reply:
It was generated when i tried to run locally the admission hooks binary i just didn't see that it was added, related to https://github.com/openshift/assisted-service/pull/3406#issuecomment-1055431677 . 
good catch, removed.